### ISSUE TITLE
Rotation doesn't update for <view>s

### ISSUE THREAD

**Issue by OhadRau** [bryphe/revery#73]
Whenever a <view> element is rotated, it keeps its initial position for the remainder of the runtime of the program. Can't tell if this is something on my end or if transforms just haven't been finished for <view> elements.

Really minimal example:

```
open Revery;
open Revery.Core;
open Revery.UI;

let init = app => {

  let w = App.createWindow(app, "test");

  let ui = UI.create(w);

  let textHeaderStyle = Style.make(~backgroundColor=Colors.red, ~color=Colors.white, ~fontFamily="Roboto-Regular.ttf", ~fontSize=24, ~marginHorizontal=12, ());

  let smallerTextStyle = Style.make(~backgroundColor=Colors.red, ~color=Colors.white, ~fontFamily="Roboto-Regular.ttf", ~fontSize=18, ~marginVertical=24, ());

  Window.setShouldRenderCallback(w, () => true);

  Window.setRenderCallback(w, () => {
    UI.render(ui,
        <view style=(Style.make(~position=LayoutTypes.Absolute, ~bottom=50, ~top=50, ~left=50, ~right=50, ~backgroundColor=Colors.blue, ()))>
            <view style=(Style.make(~position=LayoutTypes.Absolute, ~bottom=0, ~width=10, ~height=10, ~backgroundColor=Colors.red, ())) />
            <view style=(Style.make(~width=128, ~height=64, ~transform=[RotateX(Angle.from_radians(Time.getElapsedTime()))], ())) />
            <text style=(textHeaderStyle)>"Hello World!"</text>
            <text style=(smallerTextStyle)>"Welcome to revery"</text>
            <view style=(Style.make(~width=25, ~height=25, ~backgroundColor=Colors.green, ())) />
        </view>);
  });
};

App.start(init);
```

Note that this is the exact same example as Bin.re, just with the <image> tag changed to <view>.

(Btw thanks for this project, I've been waiting for something like this to come along for ages... would love to help out on some of the work for this library)

**Comment by bryphe** [comment]
Hey @OhadRau ! Thanks a lot for trying it out 😄 And for providing the minimal repro - very helpful.

Just tested it, and it's indeed a bug - we weren't updating the 'ViewNode' in our scene graph when the style changes. Pushed up a fix in #74

> (Btw thanks for this project, I've been waiting for something like this to come along for ages... would love to help out on some of the work for this library)

That would be great, I could definitely use help! Let me know if there is a particular aspect you're interested in or if you need any help getting going.

**Comment by OhadRau** [comment]
Awesome, thank you for the quick fix :) I was just messing around making some of the demos with open issues to get to know the library for now (I have a working wall clock @ https://github.com/OhadRau/revery/blob/master/examples/Clock.re but it's implemented super weirdly since rotations always anchor at the center). I'd be happy to try adding some features though, I might try to tackle anchor points with a PR (although idk what kind of API you'd want to have for that).
